lua/lvim/core/dap.lua

```diff
@@ -75,4 +76,62 @@ end
 --   end
 -- end
 
+M.setup_ui = function()
+  require("dapui").setup {
+    icons = { expanded = "", collapsed = "" },
+    mappings = {
+      -- Use a table to apply multiple mappings
+      expand = { "<CR>", "<2-LeftMouse>" },
+      open = "o",
+      remove = "d",
+      edit = "e",
+      repl = "r",
+      toggle = "t",
+    },
+    -- Expand lines larger than the window
+    -- Requires >= 0.7
+    expand_lines = vim.fn.has "nvim-0.7",
```

Comment:
```suggestion
    expand_lines = true,
```
0.7.2 is already a requirement, (and 0.8 for the next version )